api/v3/routes/tracks.rb

```diff
@@ -32,10 +32,23 @@ class Tracks < Core
         }
       end
 
+      get '/tracks/:id/img/:filename' do |id, filename|
+        track = find_track(id)
+
+        img = track.img("img/#{filename}")
+        unless img.exists?
+          halt 404, {
+            error: "No image %s for track '%s'" % [filename, id],
+          }.to_json
+        end
```

Comment:
That's a good question. I think it makes sense for x-api to be responsible for serving the default image instead of requiring exercism.io to ask questions about the image.

I'll add a root level img directory with the default and modify a few things?